src/components/Button/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Button.css';
+
+const Button = (props) => {
+	const styles = {
+		backgroundColor: props.color || '#ffa51fe3',
+		width: props.width ? '50%' : '25%'
+	}
+	return(
+	<button onClick={() => props.handleClick(props.name)} style={styles} className="button">{ props.name}</button>
```

Comment:
As mentioned previously, we can make use of object destructuring for `props` here:

```javascript
const { name, handleClick } = props;
```